inference-engine/src/auto_plugin/auto_plugin.hpp

```diff
@@ -72,6 +76,10 @@ class AutoInferencePlugin : public IE::InferencePluginInternal {
 
         return impl;
     }
+
+private:
+    ConfigType _autoConfig;
```

Comment:
why do we need a separate_autoConfig (we have _config in the base class?

Reply:
I think I should make the user to be responsible for this kind of useage, I will remove member and just use _config